code/RaygunLogWriter.php

```
@@ -103,6 +104,17 @@ function getClient() {
 	function setClient($client) {
 		$this->client = $client;
 	}
+
+	protected function filterSensitiveData() {
+		// Filter sensitive data out of server variables
+		$this->client->setFilterParams(array(
```

If this is called prior to `->getClient()` it'll throw an unrecoverable error and the original error will be lost.

In `getClient()` we can call `$this->client->filter($this->getFilters())` where `getFilters()` returns an array (no dependency on `$this->client` existing).

Would you be happy with that change?